### PR TITLE
[HOTFIX] Change footer content

### PR DESCRIPTION
**Pull request by goreck888**

### CODE REVIEW

**Review by kmarszalek — APPROVED** [review]

**Review by wujuu — APPROVED** [review]

File: app/views/designsystem/index.html.haml [hunk 1]
```diff
@@ -519,9 +519,10 @@
...
-          eInfraCentral and OpenAIRE-Advance projects funded by the Euopean
-        %p Union’s Horizon 2020 research and innovation programme with contribution of the European Commission
-
-        %p 2018 EOSC Portal
+        :ruby
+          text = "The EOSC Portal is operated by the EOSC Enhance (Grant Agreement no. 871160), "
+          text += "EOSC-hub (Grant Agreement no. 777536), and OpenAIRE-Advance (Grant Agreement no. 777541)"
+          text += "projects funded by the European Union’s Horizon 2020 research and innovation programme."
+          text += "For a complete list of contributors, visit "
+          _(text)
```
Comment (JanKapala) [inline app/views/designsystem/index.html.haml:527]: Less "workaround" and more convenient approach to write multilne text using gettext translations `_(...)` in hamls can be:
``` ruby
-# haml-lint:disable MultilinePipe
= _("some very very very " + |
    "very very very, " + |
    "very long text") |
-# haml-lint:enable MultilinePipe
```

File: app/views/layouts/_footer.html.haml [hunk 2]
```diff
@@ -152,12 +152,10 @@
...
-      = _("Union’s Horizon 2020 research and innovation programme with contribution of the European Commission")
-    %p.mt-2
-      = _("2018 EOSC Portal")
-
+      :ruby
+        text = "The EOSC Portal is operated by the EOSC Enhance (Grant Agreement no. 871160), "
+        text += "EOSC-hub (Grant Agreement no. 777536), and OpenAIRE-Advance (Grant Agreement no. 777541)"
+        text += "projects funded by the European Union’s Horizon 2020 research and innovation programme."
+        text += "For a complete list of contributors, visit "
+        _(text)
```
Comment (JanKapala) [inline app/views/layouts/_footer.html.haml:160]: Again, less "workaround" and more convenient approach to write multilne text using gettext translations `_(...)` in hamls can be:
``` ruby
-# haml-lint:disable MultilinePipe
= _("some very very very " + |
    "very very very, " + |
    "very long text") |
-# haml-lint:enable MultilinePipe
```

File: config/locales/views/layouts/en.yml [hunk 3]
```diff
@@ -5,9 +5,10 @@ en:
...
         <p class="mb-1">
-          The EOSC portal is been jointly developed and maintained by the EOSC-hub,
-          eInfraCentral and OpenAIRE-Advance projects funded by the European</p>
-        <p class="mb-1">
-          Union’s Horizon 2020 research and innovation programme with contribution of the European Commission</p>
+          The EOSC Portal is operated by the EOSC Enhance (Grant Agreement no. 871160),
+          EOSC-hub (Grant Agreement no. 777536), and OpenAIRE-Advance (Grant Agreement no. 777541)
+          projects funded by the European Union’s Horizon 2020 research and innovation programme.
+          For a complete list of contributors, visit
+          <a href= "https://www.eosc-portal.eu/about-eosc-portal">About EOSC Portal"</a></p>
```
Comment (JanKapala) [inline config/locales/views/layouts/en.yml:12]: No usage of gettext translations. Also href could be refactored using link_to method

Comment (goreck888) [reply config/locales/views/layouts/en.yml:12]: It's not used now, leaved just in case someone will use it or to delete in the future.

**Pull request merged**